src/components/Pages/Admin/View.vue

```diff
@@ -53,6 +97,78 @@
 </template>
 
 <script>
+import { mapGetters, mapActions } from 'vuex';
+import feathersClient from '../../../feathers-client';
+import GeogebraViews from '../../Geogebra/GeogebraViews';
+// import ToastrMixin from '@/mixins/ToastrMixin.vue';
+const api = feathersClient;
+
 export default {
+    // name: 'View',
+    data() {
+        return {
+            code: undefined,
+            groupsInClass: undefined,
+            GV: undefined,
+        };
+    },
+    // mixins: [ToastrMixin],
+    computed: {
+        ...mapGetters('classes', {
+            findClassesInStore: 'find',
```

Comment:
`findClasses` is enough. You don't have to indicate `InStore`, it's implied.

Reply:
In 'methods' there already are 'findClasses' and 'findGroups'